sklearn/lda.py

```diff
@@ -1,505 +1,15 @@
-          - 'eigen': Eigenvalue decomposition, can be combined with shrinkage.
-
-    shrinkage : string or float, optional
-        Shrinkage parameter, possible values:
-          - None: no shrinkage (default).
-          - 'auto': automatic shrinkage using the Ledoit-Wolf lemma.
-          - float between 0 and 1: fixed shrinkage parameter.
-
-        Note that shrinkage works only with 'lsqr' and 'eigen' solvers.
-
-    priors : array, optional, shape (n_classes,)
-        Class priors.
-
-    n_components : int, optional
-        Number of components (< n_classes - 1) for dimensionality reduction.
-
-    store_covariance : bool, optional
-        Additionally compute class covariance matrix (default False).
-
-    tol : float, optional
-        Threshold used for rank estimation in SVD solver.
-
-    Attributes
-    ----------
-    coef_ : array, shape (n_features,) or (n_classes, n_features)
-        Weight vector(s).
-
-    intercept_ : array, shape (n_features,)
-        Intercept term.
-
-    covariance_ : array-like, shape (n_features, n_features)
-        Covariance matrix (shared by all classes).
-
-    means_ : array-like, shape (n_classes, n_features)
-        Class means.
-
-    priors_ : array-like, shape (n_classes,)
-        Class priors (sum to 1).
-
-    scalings_ : array-like, shape (rank, n_classes - 1)
-        Scaling of the features in the space spanned by the class centroids.
-
-    xbar_ : array-like, shape (n_features,)
-        Overall mean.
-
-    classes_ : array-like, shape (n_classes,)
-        Unique class labels.
-
-    See also
-    --------
-    sklearn.qda.QDA: Quadratic discriminant analysis
-
-    Notes
-    -----
-    The default solver is 'svd'. It can perform both classification and
-    transform, and it does not rely on the calculation of the covariance
-    matrix. This can be an advantage in situations where the number of features
-    is large. However, the 'svd' solver cannot be used with shrinkage.
-
-    The 'lsqr' solver is an efficient algorithm that only works for
-    classification. It supports shrinkage.
-
-    The 'eigen' solver is based on the optimization of the between class
-    scatter to within class scatter ratio. It can be used for both
-    classification and transform, and it supports shrinkage. However, the
-    'eigen' solver needs to compute the covariance matrix, so it might not be
-    suitable for situations with a high number of features.
-
-    Examples
-    --------
-    >>> import numpy as np
-    >>> from sklearn.lda import LDA
-    >>> X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]])
-    >>> y = np.array([1, 1, 1, 2, 2, 2])
-    >>> clf = LDA()
-    >>> clf.fit(X, y)
-    LDA(n_components=None, priors=None, shrinkage=None, solver='svd',
-      store_covariance=False, tol=0.0001)
-    >>> print(clf.predict([[-0.8, -1]]))
-    [1]
-    """
+class LDA(LinearDiscriminantAnalysis):
```

Comment:
Now that deprecation is in the module, just write `pass` here. The only reason to include the class definition at all is if any code relies on the `__class__.__name__`. Otherwise, you could just use `from .discriminant_analysis import LinearDiscriminantAnalysis as LDA`.